code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -23,6 +23,8 @@ pub const WASM_BINARY_V2: Option<&[u8]> = Some(include_bytes!(env!("PICASSO_RUNT
 #[cfg(not(feature = "builtin-wasm"))]
 pub const WASM_BINARY_V2: Option<&[u8]> = None;
 
+extern crate alloc;
```

Comment:
isn't extern crate deprecated, and we can just refer to alloc directly?

Reply:
I thought the same, but

```
error[E0433]: failed to resolve: use of undeclared crate or module `alloc`
  --> parachain/runtime/picasso/src/lib.rs:37:5
   |
37 | use alloc::string::String;
   |     ^^^^^ use of undeclared crate or module `alloc`
   |
   = help: add `extern crate alloc` to use the `alloc` crate
   ```